<!DOCTYPE HTML>
<html lang="en">


<head><meta name="generator" content="Hexo 3.9.0">
    <meta charset="utf-8">
    <meta name="keywords" content="Protein Structure Modelling, MD">
    <meta name="description" content="
Protein Structure Modelling
Prediction of the three dimensional structure of a target protein from the amino acid seque">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no">
    <meta name="renderer" content="webkit|ie-stand|ie-comp">
    <meta name="mobile-web-app-capable" content="yes">
    <meta name="format-detection" content="telephone=no">
    <meta name="apple-mobile-web-app-capable" content="yes">
    <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent">
    <title>Protein Structure Modelling | XINJIE</title>
    <link rel="icon" type="image/png" href="/favicon.png">

    <link rel="stylesheet" type="text/css" href="/libs/awesome/css/all.css">
    <link rel="stylesheet" type="text/css" href="/libs/materialize/materialize.min.css">
    <link rel="stylesheet" type="text/css" href="/libs/aos/aos.css">
    <link rel="stylesheet" type="text/css" href="/libs/animate/animate.min.css">
    <link rel="stylesheet" type="text/css" href="/libs/lightGallery/css/lightgallery.min.css">
    <link rel="stylesheet" type="text/css" href="/css/matery.css">
    <link rel="stylesheet" type="text/css" href="/css/my.css">

    <script src="/libs/jquery/jquery-2.2.0.min.js"></script>
    
<link rel="stylesheet" href="/css/prism-tomorrow.css" type="text/css"></head>


<body>
    <header class="navbar-fixed">
    <nav id="headNav" class="bg-color nav-transparent">
        <div id="navContainer" class="nav-wrapper head-container">
            <div class="brand-logo">
                <a href="/" class="waves-effect waves-light">
                    
                    <img src="/medias/logo.png" class="logo-img" alt="LOGO">
                    
                    <span class="logo-span">XINJIE</span>
                </a>
            </div>
            

<a href="#" data-target="mobile-nav" class="sidenav-trigger button-collapse"><i class="fas fa-bars"></i></a>
<ul class="right nav-menu">
  
  <li class="hide-on-med-and-down nav-item">
    
    <a href="/" class="waves-effect waves-light">
      
      <i class="fas fa-home" style="zoom: 0.6;"></i>
      
      <span>Home</span>
    </a>
    
  </li>
  
  <li class="hide-on-med-and-down nav-item">
    
    <a href="/tags" class="waves-effect waves-light">
      
      <i class="fas fa-tags" style="zoom: 0.6;"></i>
      
      <span>Tags</span>
    </a>
    
  </li>
  
  <li class="hide-on-med-and-down nav-item">
    
    <a href="/categories" class="waves-effect waves-light">
      
      <i class="fas fa-bookmark" style="zoom: 0.6;"></i>
      
      <span>Categories</span>
    </a>
    
  </li>
  
  <li class="hide-on-med-and-down nav-item">
    
    <a href="/galleries" class="waves-effect waves-light">
      
      <i class="fas fa-images" style="zoom: 0.6;"></i>
      
      <span>Galleries</span>
    </a>
    
  </li>
  
  <li class="hide-on-med-and-down nav-item">
    
    <a href="/about" class="waves-effect waves-light">
      
      <i class="fas fa-user-circle" style="zoom: 0.6;"></i>
      
      <span>About</span>
    </a>
    
  </li>
  
  <li class="hide-on-med-and-down nav-item">
    
    <a href="/friends" class="waves-effect waves-light">
      
      <i class="fas fa-address-book" style="zoom: 0.6;"></i>
      
      <span>Friends</span>
    </a>
    
  </li>
  
  <li>
    <a href="#searchModal" class="modal-trigger waves-effect waves-light">
      <i id="searchIcon" class="fas fa-search" title="Search" style="zoom: 0.85;"></i>
    </a>
  </li>
</ul>

<div id="mobile-nav" class="side-nav sidenav">

    <div class="mobile-head bg-color">
        
        <img src="/medias/logo.png" class="logo-img circle responsive-img">
        
        <div class="logo-name">XINJIE</div>
        <div class="logo-desc">
            
            (●&#39;◡&#39;●)
            
        </div>
    </div>

    

    <ul class="menu-list mobile-menu-list">
        
        <li class="m-nav-item">
	  
		<a href="/" class="waves-effect waves-light">
			
			    <i class="fa-fw fas fa-home"></i>
			
			Home
		</a>
          
        </li>
        
        <li class="m-nav-item">
	  
		<a href="/tags" class="waves-effect waves-light">
			
			    <i class="fa-fw fas fa-tags"></i>
			
			Tags
		</a>
          
        </li>
        
        <li class="m-nav-item">
	  
		<a href="/categories" class="waves-effect waves-light">
			
			    <i class="fa-fw fas fa-bookmark"></i>
			
			Categories
		</a>
          
        </li>
        
        <li class="m-nav-item">
	  
		<a href="/galleries" class="waves-effect waves-light">
			
			    <i class="fa-fw fas fa-images"></i>
			
			Galleries
		</a>
          
        </li>
        
        <li class="m-nav-item">
	  
		<a href="/about" class="waves-effect waves-light">
			
			    <i class="fa-fw fas fa-user-circle"></i>
			
			About
		</a>
          
        </li>
        
        <li class="m-nav-item">
	  
		<a href="/friends" class="waves-effect waves-light">
			
			    <i class="fa-fw fas fa-address-book"></i>
			
			Friends
		</a>
          
        </li>
        
        
        <li><div class="divider"></div></li>
        <li>
            <a href="https://github.com/pandatt66/pandatt66.github.io" class="waves-effect waves-light" target="_blank">
                <i class="fab fa-github-square fa-fw"></i>pandatt
            </a>
        </li>
        
    </ul>
</div>


        </div>

        
            <style>
    .nav-transparent .github-corner {
        display: none !important;
    }

    .github-corner {
        position: absolute;
        z-index: 10;
        top: 0;
        right: 0;
        border: 0;
        transform: scale(1.1);
    }

    .github-corner svg {
        color: #0f9d58;
        fill: #fff;
        height: 64px;
        width: 64px;
    }

    .github-corner:hover .octo-arm {
        animation: a 0.56s ease-in-out;
    }

    .github-corner .octo-arm {
        animation: none;
    }

    @keyframes a {
        0%,
        to {
            transform: rotate(0);
        }
        20%,
        60% {
            transform: rotate(-25deg);
        }
        40%,
        80% {
            transform: rotate(10deg);
        }
    }
</style>

<a href="https://github.com/pandatt66/pandatt66.github.io" class="github-corner tooltipped hide-on-med-and-down" target="_blank"
   data-tooltip="pandatt" data-position="left" data-delay="50">
    <svg viewBox="0 0 250 250" aria-hidden="true">
        <path d="M0,0 L115,115 L130,115 L142,142 L250,250 L250,0 Z"></path>
        <path d="M128.3,109.0 C113.8,99.7 119.0,89.6 119.0,89.6 C122.0,82.7 120.5,78.6 120.5,78.6 C119.2,72.0 123.4,76.3 123.4,76.3 C127.3,80.9 125.5,87.3 125.5,87.3 C122.9,97.6 130.6,101.9 134.4,103.2"
              fill="currentColor" style="transform-origin: 130px 106px;" class="octo-arm"></path>
        <path d="M115.0,115.0 C114.9,115.1 118.7,116.5 119.8,115.4 L133.7,101.6 C136.9,99.2 139.9,98.4 142.2,98.6 C133.8,88.0 127.5,74.4 143.8,58.0 C148.5,53.4 154.0,51.2 159.7,51.0 C160.3,49.4 163.2,43.6 171.4,40.1 C171.4,40.1 176.1,42.5 178.8,56.2 C183.1,58.6 187.2,61.8 190.9,65.4 C194.5,69.0 197.7,73.2 200.1,77.6 C213.8,80.2 216.3,84.9 216.3,84.9 C212.7,93.1 206.9,96.0 205.4,96.6 C205.1,102.4 203.0,107.8 198.3,112.5 C181.9,128.9 168.3,122.5 157.7,114.1 C157.9,116.9 156.7,120.9 152.7,124.9 L141.0,136.5 C139.8,137.7 141.6,141.9 141.8,141.8 Z"
              fill="currentColor" class="octo-body"></path>
    </svg>
</a>
        
    </nav>

</header>

    



<div class="bg-cover pd-header post-cover" style="background-image: url('/images/1004.jpg')">
    <div class="container" style="right: 0px;left: 0px;">
        <div class="row">
            <div class="col s12 m12 l12">
                <div class="brand">
                    <div class="description center-align post-title">
                        Protein Structure Modelling
                    </div>
                </div>
            </div>
        </div>
    </div>
</div>




<main class="container content">

    
    <!-- 文章内容详情 -->
<div id="artDetail">
    <div class="card">
        <div class="card-content article-info">
            <div class="row tag-cate">
                <div class="col s7">
                    
                    <div class="article-tag">
                        
                            <a href="/tags/Homology-Modelling/">
                                <span class="chip bg-color">Homology Modelling</span>
                            </a>
                        
                            <a href="/tags/Protein-structure-prediction/">
                                <span class="chip bg-color">Protein structure prediction</span>
                            </a>
                        
                            <a href="/tags/Swiss-Model/">
                                <span class="chip bg-color">Swiss Model</span>
                            </a>
                        
                    </div>
                    
                </div>
                <div class="col s5 right-align">
                    
                    <div class="post-cate">
                        <i class="fas fa-bookmark fa-fw icon-category"></i>
                        
                            <a href="/categories/Drug-discovery/" class="post-category">
                                Drug discovery
                            </a>
                        
                    </div>
                    
                </div>
            </div>

            <div class="post-info">
                <div class="post-date info-break-policy">
                    <i class="far fa-calendar-minus fa-fw"></i>Publish Date:&nbsp;&nbsp;
                    2019-11-20
                </div>
                
                
                <div class="post-date info-break-policy">
                    <i class="far fa-calendar-check fa-fw"></i>Update Date:&nbsp;&nbsp;
                    2019-11-28
                </div>
                

                
                    
                    <div class="info-break-policy">
                        <i class="far fa-file-word fa-fw"></i>Word Count:&nbsp;&nbsp;
                        1.9k
                    </div>
                    

                    
                    <div class="info-break-policy">
                        <i class="far fa-clock fa-fw"></i>Read Times:&nbsp;&nbsp;
                        11 Min
                    </div>
                    
                
				
				
            </div>
        </div>
        <hr class="clearfix">
        <div class="card-content article-card-content">
            <div id="articleContent">
                <hr>
<h1 id="Protein-Structure-Modelling"><a href="#Protein-Structure-Modelling" class="headerlink" title="Protein Structure Modelling"></a>Protein Structure Modelling</h1><blockquote>
<p>Prediction of the three dimensional structure of a target protein from the amino acid sequence of a homologous (template) protein for which an X-ray or NMR structure is available.</p>
</blockquote>
<h2 id="Three-approaches-to-structure-prediction"><a href="#Three-approaches-to-structure-prediction" class="headerlink" title="Three approaches to structure prediction:"></a>Three approaches to structure prediction:</h2><ol>
<li><p>Ab initio(de novo) prediction</p>
<ul>
<li>Predict the 3D structure from “first principles”, based on energetic or statistical principles. It cost large computational power and time.<blockquote>
<p><em>ex.QUARK(<a href="https://omictools.com/quark-tool" target="_blank" rel="noopener">https://omictools.com/quark-tool</a>)</em><br>-A computer algorithm for ab initio protein folding and protein structure prediction, which aims to construct the correct protein 3D model from amino acid sequence only.<br>-QUARK models are built from small fragments (1-20 residues long) by replica-exchange Monte Carlo simulation under the guide of an atomic-level knowledge-based force field.<br>TM-score: &gt;0.5 reliable<br>&lt;0.17 not reliable</p>
</blockquote>
</li>
</ul>
</li>
<li><p>Sequence- Structure Threading</p>
<ul>
<li>Given the sequence, and a set of folds observed in PDB, see if any of the sequences could adopt one the known folds.<blockquote>
<p><em>ex.I-TASSER(<a href="https://zhanglab.ccmb.med.umich.edu/I-TASSER/" target="_blank" rel="noopener">https://zhanglab.ccmb.med.umich.edu/I-TASSER/</a>)</em><br>I-TASSER (Iterative Threading ASSEmbly Refinement) is a hierarchical approach to protein structure and function prediction.</p>
</blockquote>
</li>
<li>It first identifies structural templates from the PDB by multiple threading approach LOMETS, with full-length atomic models constructed by iterative template-based fragment assembly simulations.<br>Other tools:<a href="https://omictools.com/raptorx-tool" target="_blank" rel="noopener">RaptorX</a>,<a href="https://omictools.com/foldx-tool" target="_blank" rel="noopener">FoldX</a><br>C-score:[-5,2],the higher the better.</li>
</ul>
</li>
<li><p><strong>Homology Modelling</strong><br>If two proteins are homologous (which can be assumed with 50% similarity), then in almost all cases they will have similar folds.<br>You only use this method when a given sequence with homology (&gt; 30%) to a known structure as template to create a 3D model from the sequence.<br><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi" target="_blank" rel="noopener">BLAST</a> finds regions of similarity between biological sequences.</p>
<blockquote>
<p><a href="https://www.schrodinger.com/training/building-homology-models-prime-case-study-factor-xa-workshop-tutorial-1" target="_blank" rel="noopener">Schrodinger Prime</a>:”Building Homology Models with Prime: A Case Study with Factor Xa Workshop Tutorial”.<br><a href="https://salilab.org/modeller/" target="_blank" rel="noopener">Modeller</a>:Python script open software source<br><a href="https://swissmodel.expasy.org/" target="_blank" rel="noopener">Swiss Model</a>: Homology Modelling Of Superoxide Dismutase [Cu-Zn]: Step by step example on how to build a homology model using SWISS-MODEL(Qmean4:[0,1],higher the better)</p>
</blockquote>
</li>
</ol>
<p><img src="https://miro.medium.com/max/1595/0*JYVw2gz5GZHeLld8.jpg" alt="Overall approach"></p>
<h1 id="How-homology-modeling-works"><a href="#How-homology-modeling-works" class="headerlink" title="How homology modeling works?"></a>How homology modeling works?</h1><p><img src="https://slideplayer.com/slide/5692232/18/images/10/Homology+Modeling+Most+of+the+steps+can+be+automated%21%21+Very.jpg" alt></p>
<h3 id="Target-Template-Sequence-Alignment"><a href="#Target-Template-Sequence-Alignment" class="headerlink" title="Target-Template Sequence Alignment"></a>Target-Template Sequence Alignment</h3><p>Absolutely Critical:<br>– Sequence alignment is the bottleneck of the modeling process<br>– No comparative modeling scheme can recover from an incorrect alignment.<br>How does one find template(s)?<br>The simplest template determination approaches use fairly common database searching methods (i.e., BLAST and FASTA).<br>In slightly more difficult cases, multiple sequence alignment and profile-based methods might be used to identify and better align the template to the target sequence.</p>
<p>When multiple targets are identified, there are a variety of ways of determining the best — this is a very important step.<br>Key factors to consider include:<br>coverage,<br>sequence similarity/phylogenetic clustering,<br>matching of target predicted secondary structure with observed template secondary structure,<br>structure quality (resolution, R-factor, etc.),<br>known functional relationships, etc.</p>
<h3 id="Backbone-Model-Generation"><a href="#Backbone-Model-Generation" class="headerlink" title="Backbone Model Generation"></a>Backbone Model Generation</h3><p>For most of the model, creating the backbone structure with traditional homology modeling protocol is trivial (simply copy the coordinates from one to template to the model!). If there is a match within the alignment, the coordinates of the sidechain can be copied as well.<br>More recent methods attempt to use multiple structural templates (e.g. if one template has good overlap in one area, while the other has better overlap elsewhere).</p>
<p>The program SEGMOD builds the model structure using a hexapeptide fragment library. The model structure is built based on a series of these fragments.<br>The widely used program MODELER generates a series of distance constraints from the template structure, and then build a model using these restraints in much the same way that is done with NMR structure determination.<br>One of the advantages of using satisfaction of spatial restraints method is that it can incorporate various restraints from experiments, such as NMR experiments, site-directed mutagenesis and cross-linking experiments.</p>
<h3 id="Loop-Modeling"><a href="#Loop-Modeling" class="headerlink" title="Loop Modeling"></a>Loop Modeling</h3><p>Modeling loops that lack coverage within the template is extremely difficult, yet common due to:<br>Template structure is not well resolved.<br>Sequence divergence.<br>Insertions/Deletions<br>To make things worse, loop regions vary significantly between model and template even when complete coverage is present.<br>Surface loops tend to be involved in crystal contacts, leading to significant conformational changes dependent upon the unit cell.<br>The exchange of a small to bulky sidechain underneath the loop (within the core) can “push” it aside.<br>Also, remember that loop regions are generally floppy and fluctuate constantly, meaning a fixed conformation may have little biological meaning.</p>
<p>20 Loop Modeling Methods Knowledge-based: Energy-based:<br>Find matching loops with the right number of residues and matching endpoints within the PDB.<br>In particularly difficult cases (loops longer than ~8 residues), chain fragments together. Based on the premise that irregular substructures are built from combinations of Small Standard Structures.<br>Energy-based:<br>Generate random loops of right length and endpoints. Evaluate resultant structure with some sort of energy function.</p>
<h3 id="Side-chain-Modeling-Packing"><a href="#Side-chain-Modeling-Packing" class="headerlink" title="Side-chain Modeling/Packing"></a>Side-chain Modeling/Packing</h3><p>Some sort of knowledge-based rotamer library from high-resolution structures is used.<br>Combinatorial explosion:<br>Intuitively, it makes sense that the conformation of one will affect the conformations of others.<br>Fortunately, rotamer space is not limitless.<br>Assuming, fiver rotamers per residue, there is still 5100 different combinations to score within a 100 amino acid protein.<br>Solutions:<br>Certain backbone conformations strongly favor certain rotamers, meaning the others can be ignored.<br>More rigid residues can be modeled first, and the more flexible (larger rotamer space) can be modeled subsequently. The advantage of this is that the more rigid residue limits the space that must be explored by the flexible one.<br>Nature picks rotamer conformations that maximize packing (minimize voids) and the number of interactions with other groups (i.e. H-bonds, salt bridges, disulfide bonds, etc.).</p>
<h3 id="Model-Optimization"><a href="#Model-Optimization" class="headerlink" title="Model Optimization"></a>Model Optimization</h3><p>Sometimes the rotamers are predicted based on incorrect backbone or incorrect prediction. Such cases modeling programs either restrain the atom positions and/or apply only a few hundred steps of energy minimization to get an accurate value. This accuracy can be achieved by 2 ways.</p>
<ol>
<li>Quantum force field: To handle large molecules efficiently force field can be used, energies are therefore normally expressed as a function of the positions of the atomic nuclei only. Van der Waals forces are, for example, so difficult to treat, that they must often be completely omitted. While providing more accurate electrostatics, the overall precision achieved is still about the same as in the classical force fields.</li>
</ol>
<ol start="2">
<li>Self-parametrizing force fields: The precision of a force field depends to a large extent on its parameters (e.g., Van der Waals radii, atomic charges). These parameters are usually obtained from quantum chemical calculations on small molecules and fitting to experimental data, following elaborate rules (Wang, Cieplak, and Kollman, 2000). By applying the force field to proteins, one implicitly assumes that a peptide chain is just the sum of its individual small molecule building blocks—the amino acids. To increase the precision of the force field, the following steps can be used. Take initial parameters (for example, from an existing force field), change a parameter randomly, energy minimize models, see if the result improved, keep the new force field if yes, otherwise go back to the previous force field.</li>
</ol>
<p>The models we obtain may contain errors. These errors mainly depend upon two values.</p>
<ol>
<li>The percentage identity between the template and the target.</li>
</ol>
<p>If the value is &gt; 90% then accuracy can be compared to crystallography, except for a few individual side chains. If its value ranges between 50-90 % r.m.s.d. error can be as large as 1.5 Å, with considerably more errors. If the value is &lt;25% the alignment turns out to be difficult for homology modeling, often leading to quite larger errors.</p>
<ol start="2">
<li>The number of errors in the template.</li>
</ol>
<p>Errors in a model become less of a problem if they can be localized. Therefore, an essential step in the homology modeling process is the verification of the model. The errors can be estimated by calculating the model’s energy based on a force field. This method checks to see if the bond lengths and angles are in a normal range. However, this method cannot judge if the model is correctly folded. The 3D distribution functions can also easily identify misfolded proteins and are good indicators of local model building problems</p>
<h2 id="Automatic-modelling-–-Swiss-Model"><a href="#Automatic-modelling-–-Swiss-Model" class="headerlink" title="Automatic modelling – Swiss Model"></a>Automatic modelling – Swiss Model</h2><ul>
<li>Modelling → myWorkspace (Start a new modelling project.)<ul>
<li>You can either paste the protein sequence or provide the UniprotKB AC (P61851) of your target sequence in the input form.</li>
</ul>
</li>
<li>Click “Search for Templates”</li>
<li>Template Results<ul>
<li>A checkbox to select and visualise the template in the 3D panel</li>
<li>The SMTL ID of the template and a link to the SWISS-MODEL Template Library page associated to that SMTL entry</li>
<li>The protein name of the template</li>
<li>The GMQE (Global Model Quality Estimation)</li>
<li>The QSQE (Quaternary Structure Quality Estimation)</li>
<li>The target–template sequence identity</li>
<li>A clickable icon to expand the box with the description of the template</li>
</ul>
</li>
</ul>
<p>Typically, due to their higher variability in terms of sequence and structure compared to the rest of the protein, loops are the regions that are the most susceptible to modelling errors.<br>On the other hand, they are often involved in protein function, hence accurately modelling their structure is of crucial importance.</p>
<p>Are these loops important for the function of the protein?</p>
<ul>
<li>As expected, the regions that show most of the structural differences correspond to positions in the alignment where amino acid substitutitons and indels are present.<br>Even if these differences seem to have little or no impact on the overall structure of the protein, they can still play an important role for the function of the protein, i.e. ligand- and/or substrate-binding.</li>
</ul>
<p>After you have selected the template(s), click the ‘Build Model’ button to run the modelling job.</p>
<p>Compared to the sequence of the target, they show some insertions and deletions as well as some amino acid substitution.<br>By clicking on the cog icon, you can change the colouring scheme of the alignement by selecting one of the many available options. Let’s select the Clustal colouring scheme.</p>
<h1 id="Model-assessment"><a href="#Model-assessment" class="headerlink" title="Model assessment"></a>Model assessment</h1><p>(Swiss model:<a href="https://swissmodel.expasy.org/assess/help" target="_blank" rel="noopener">https://swissmodel.expasy.org/assess/help</a>)<br>SAVES server (<a href="http://nihserver.mbi.ucla.edu/SAVES/" target="_blank" rel="noopener">http://nihserver.mbi.ucla.edu/SAVES/</a>) from NIH MBI Laboratory is mostly useful for analysis.<br>This metaserver runs 6 programs(PROCHECK、WHAT_CHECK、ERRAT、VERIFY_3D、PROVE、CRYST1) for checking and validating protein structures during and after model refinement.<br>The Ramachandran plot function in the Model Panel plots the distribution of amino acid backbone conformations in peptide and protein structures.<br><img src="http://www.cryst.bbk.ac.uk/PPS95/course/3_geometry/rama.gif" alt><br><img src="https://pic3.zhimg.com/80/v2-993139db156f9fb29ea395fc1dc77f26_hd.jpg" alt><br>The two torsion angles of the polypeptide chain, also called Ramachandran angles, describe the rotations of the polypeptide backbone around the bonds between N-Cα (called Phi, φ) and Cα-C (called Psi, ψ). The Ramachandran plot provides an easy way to view the distribution of torsion angles of a protein structure. It also provides an overview of allowed and disallowed regions of torsion angle values, serving as an important factor in the assessment of the quality of protein three-dimensional structures.</p>
<h3 id="Q-What-to-Do-with-Your-Protein-Model"><a href="#Q-What-to-Do-with-Your-Protein-Model" class="headerlink" title="Q:What to Do with Your Protein Model?"></a>Q:What to Do with Your Protein Model?</h3><p>Molecular Interaction Studies – Molecular Docking.<br>Computer-aided drug discovery.<br>Improve the results of Quantitative Structure-Activity Relationship (QSAR) techniques.</p>
<p>Sources:<br><a href="https://medium.com/@HeleneOMICtools/a-guide-for-protein-structure-prediction-methods-and-software-916a2f718cfe" target="_blank" rel="noopener">https://medium.com/@HeleneOMICtools/a-guide-for-protein-structure-prediction-methods-and-software-916a2f718cfe</a><br><a href="https://zhanglab.ccmb.med.umich.edu/papers/2010_3.pdf" target="_blank" rel="noopener">I-TASSER: a unified platform for automated protein structure and function prediction</a><br><a href="https://vlab.amrita.edu/?sub=3&brch=277&sim=1492&cnt=1" target="_blank" rel="noopener">Predict the structure of protein-Homology Modeling</a><br><a href="https://zhuanlan.zhihu.com/p/45041082" target="_blank" rel="noopener">https://zhuanlan.zhihu.com/p/45041082</a><br><a href="https://vlab.amrita.edu/?sub=3&amp;brch=275&amp;sim=1481&amp;cnt=1" target="_blank" rel="noopener">https://vlab.amrita.edu/?sub=3&amp;brch=275&amp;sim=1481&amp;cnt=1</a><br><a href="https://en.m.wikibooks.org/wiki/Structural_Biochemistry/Proteins/Ramachandran_Plot" target="_blank" rel="noopener">https://en.m.wikibooks.org/wiki/Structural_Biochemistry/Proteins/Ramachandran_Plot</a></p>

            </div>
            <hr/>

            

    <div class="reprint" id="reprint-statement">
        
            <div class="reprint__author">
                <span class="reprint-meta" style="font-weight: bold;">
                    <i class="fas fa-user">
                        Author:
                    </i>
                </span>
                <span class="reprint-info">
                    <a href="https://pandatt66.github.io" rel="external nofollow noreferrer">Xinjie</a>
                </span>
            </div>
            <div class="reprint__type">
                <span class="reprint-meta" style="font-weight: bold;">
                    <i class="fas fa-link">
                        Link:
                    </i>
                </span>
                <span class="reprint-info">
                    <a href="https://pandatt66.github.io/2019/11/20/1Drug discovery/1004Swissmodel/">https://pandatt66.github.io/2019/11/20/1Drug discovery/1004Swissmodel/</a>
                </span>
            </div>
            <div class="reprint__notice">
                <span class="reprint-meta" style="font-weight: bold;">
                    <i class="fas fa-copyright">
                        Reprint policy:
                    </i>
                </span>
                <span class="reprint-info">
                    All articles in this blog are used except for special statements
                    <a href="https://creativecommons.org/licenses/by/4.0/deed.zh" rel="external nofollow noreferrer" target="_blank">CC BY 4.0</a>
                    reprint polocy. If reproduced, please indicate source
                    <a href="https://pandatt66.github.io" target="_blank">Xinjie</a>
                    !
                </span>
            </div>
        
    </div>

    <script async defer>
      document.addEventListener("copy", function (e) {
        let toastHTML = '<span>Copied successfully, please follow the reprint policy of this article</span><button class="btn-flat toast-action" onclick="navToReprintStatement()" style="font-size: smaller">more</a>';
        M.toast({html: toastHTML})
      });

      function navToReprintStatement() {
        $("html, body").animate({scrollTop: $("#reprint-statement").offset().top - 80}, 800);
      }
    </script>



            <div class="tag_share" style="display: block;">
                <div class="post-meta__tag-list" style="display: inline-block;">
                    
                        <div class="article-tag">
                            
                                <a href="/tags/Homology-Modelling/">
                                    <span class="chip bg-color">Homology Modelling</span>
                                </a>
                            
                                <a href="/tags/Protein-structure-prediction/">
                                    <span class="chip bg-color">Protein structure prediction</span>
                                </a>
                            
                                <a href="/tags/Swiss-Model/">
                                    <span class="chip bg-color">Swiss Model</span>
                                </a>
                            
                        </div>
                    
                </div>
                <div class="post_share" style="zoom: 80%; width: fit-content; display: inline-block; float: right; margin: -0.15rem 0;">
                    <link rel="stylesheet" type="text/css" href="/libs/share/css/share.min.css">

<div id="article-share">
    
    
    <div class="social-share" data-sites="twitter,facebook,google,wechat,weibo,douban" data-wechat-qrcode-helper="<p>微信扫一扫即可分享！</p>"></div>
    <script src="/libs/share/js/social-share.min.js"></script>
    

    

</div>

                </div>
            </div>
            
        </div>
    </div>

    
        <link rel="stylesheet" href="/libs/gitalk/gitalk.css">
<link rel="stylesheet" href="/css/my-gitalk.css">

<div class="card gitalk-card" data-aos="fade-up">
    <div id="gitalk-container" class="card-content"></div>
</div>

<script src="/libs/gitalk/gitalk.min.js"></script>
<script>
    let gitalk = new Gitalk({
        clientID: '915ede58ab44888a86d9',
        clientSecret: '3a674c1c606802816b6f99c605dbc60c6525f828',
        repo: 'gitment',
        owner: 'pandatt66',
        admin: "pandatt66",
        id: '2019-11-20T11-13-11',
        distractionFreeMode: false  // Facebook-like distraction free mode
    });
    gitalk.render('gitalk-container');
</script>
    

    

    

    

    

    

<article id="prenext-posts" class="prev-next articles">
    <div class="row article-row">
        
        <div class="article col s12 m6" data-aos="fade-up">
            <div class="article-badge left-badge text-color">
                <i class="fas fa-chevron-left"></i>&nbsp;Previous</div>
            <div class="card">
                <a href="/2019/11/22/1Drug discovery/1003HTVS/">
                    <div class="card-image">
                        
                        <img src="/images/1003.jpg" class="responsive-img" alt="Virtual Screening">
                        
                        <span class="card-title">Virtual Screening</span>
                    </div>
                </a>
                <div class="card-content article-content">
                    <div class="summary block-with-text">
                        
                            HTVS
                        
                    </div>
                    <div class="publish-info">
                        <span class="publish-date">
                            <i class="far fa-clock fa-fw icon-date"></i>2019-11-22
                        </span>
                        <span class="publish-author">
                            
                            <i class="fas fa-bookmark fa-fw icon-category"></i>
                            
                            <a href="/categories/Drug-discovery/" class="post-category">
                                    Drug discovery
                                </a>
                            
                            
                        </span>
                    </div>
                </div>
                
                <div class="card-action article-tags">
                    
                    <a href="/tags/Virtual-Screening/">
                        <span class="chip bg-color">Virtual Screening</span>
                    </a>
                    
                    <a href="/tags/Schrodinger/">
                        <span class="chip bg-color">Schrödinger</span>
                    </a>
                    
                </div>
                
            </div>
        </div>
        
        
        <div class="article col s12 m6" data-aos="fade-up">
            <div class="article-badge right-badge text-color">
                Next&nbsp;<i class="fas fa-chevron-right"></i>
            </div>
            <div class="card">
                <a href="/2019/11/09/1Drug discovery/1002AD/">
                    <div class="card-image">
                        
                        <img src="/images/1002.jpg" class="responsive-img" alt="Alzheimer’s disease (AD)">
                        
                        <span class="card-title">Alzheimer’s disease (AD)</span>
                    </div>
                </a>
                <div class="card-content article-content">
                    <div class="summary block-with-text">
                        
                            Alzheimer’s disease
                        
                    </div>
                    <div class="publish-info">
                            <span class="publish-date">
                                <i class="far fa-clock fa-fw icon-date"></i>2019-11-09
                            </span>
                        <span class="publish-author">
                            
                            <i class="fas fa-bookmark fa-fw icon-category"></i>
                            
                            <a href="/categories/Neuroscience/" class="post-category">
                                    Neuroscience
                                </a>
                            
                            
                        </span>
                    </div>
                </div>
                
                <div class="card-action article-tags">
                    
                    <a href="/tags/Alzheimer’s-disease-AD/">
                        <span class="chip bg-color">Alzheimer’s disease (AD)</span>
                    </a>
                    
                </div>
                
            </div>
        </div>
        
    </div>
</article>

</div>



<!-- 代码块功能依赖 -->
<script type="text/javascript" src="/libs/codeBlock/codeBlockFuction.js"></script>

<!-- 代码语言 -->

<script type="text/javascript" src="/libs/codeBlock/codeLang.js"></script>

    
<!-- 代码块复制 -->

<script type="text/javascript" src="/libs/codeBlock/codeCopy.js"></script>


<!-- 代码块收缩 -->

<script type="text/javascript" src="/libs/codeBlock/codeShrink.js"></script>


<!-- 代码块折行 -->

<style type="text/css">
code[class*="language-"], pre[class*="language-"] { white-space: pre !important; }
</style>


    

</main>



    <footer class="page-footer bg-color">
    <div class="container row center-align">
        <div class="col s12 m8 l8 copy-right">
            &copy; 2019 Xinjie Tong. All Rights Reserved.

            
            &nbsp;<i class="fa fa-area-chart"></i>&nbsp;Total words:&nbsp;
            <span class="white-color">5.2k</span>
            

            <br>
            <span id="sitetime"></span>

            
            
        </div>
        <div class="col s12 m4 l4 social-link ">














</div>
    </div>
</footer>

<div class="progress-bar"></div>

<!-- 不蒜子计数初始值纠正 -->
<script>
    $(document).ready(function () {
        var int = setInterval(fixCount, 50);
        var pvcountOffset = 80000;
        var uvcountOffset = 20000;
        function fixCount() {
            if (document.getElementById("busuanzi_container_site_pv").style.display != "none") {
                $("#busuanzi_value_site_pv").html(parseInt($("#busuanzi_value_site_pv").html()) + pvcountOffset);
                clearInterval(int);
            }
            if ($("#busuanzi_container_site_pv").css("display") != "none") {
                $("#busuanzi_value_site_uv").html(parseInt($("#busuanzi_value_site_uv").html()) + uvcountOffset); // 加上初始数据 
                clearInterval(int);
            }
        }
    });
</script>

<script language=javascript>
    function siteTime() {
        window.setTimeout("siteTime()", 1000);
        var seconds = 1000;
        var minutes = seconds * 60;
        var hours = minutes * 60;
        var days = hours * 24;
        var years = days * 365;
        var today = new Date();
        var todayYear = today.getFullYear();
        var todayMonth = today.getMonth() + 1;
        var todayDate = today.getDate();
        var todayHour = today.getHours();
        var todayMinute = today.getMinutes();
        var todaySecond = today.getSeconds();
        /* Date.UTC() -- 返回date对象距世界标准时间(UTC)1970年1月1日午夜之间的毫秒数(时间戳)
        year - 作为date对象的年份，为4位年份值
        month - 0-11之间的整数，做为date对象的月份
        day - 1-31之间的整数，做为date对象的天数
        hours - 0(午夜24点)-23之间的整数，做为date对象的小时数
        minutes - 0-59之间的整数，做为date对象的分钟数
        seconds - 0-59之间的整数，做为date对象的秒数
        microseconds - 0-999之间的整数，做为date对象的毫秒数 */
        var t1 = Date.UTC(2019, 10, 27, 00, 00, 00); //北京时间2018-2-13 00:00:00
        var t2 = Date.UTC(todayYear, todayMonth, todayDate, todayHour, todayMinute, todaySecond);
        var diff = t2 - t1;
        var diffYears = Math.floor(diff / years);
        var diffDays = Math.floor((diff / days) - diffYears * 365);
        var diffHours = Math.floor((diff - (diffYears * 365 + diffDays) * days) / hours);
        var diffMinutes = Math.floor((diff - (diffYears * 365 + diffDays) * days - diffHours * hours) / minutes);
        var diffSeconds = Math.floor((diff - (diffYears * 365 + diffDays) * days - diffHours * hours - diffMinutes * minutes) / seconds);
        document.getElementById("sitetime").innerHTML = "本站已运行 " + diffYears + " 年 " + diffDays + " 天 " + diffHours + " 小时 " + diffMinutes + " 分钟 " + diffSeconds + " 秒";
    }/*因为建站时间还没有一年，就将之注释掉了。需要的可以取消*/
    siteTime();

<span class="post-count">5.2k words altogether</span>
</script>

    <!-- 搜索遮罩框 -->
<div id="searchModal" class="modal">
    <div class="modal-content">
        <div class="search-header">
            <span class="title"><i class="fas fa-search"></i>&nbsp;&nbsp;Search</span>
            <input type="search" id="searchInput" name="s" placeholder="Please enter a search keyword"
                   class="search-input">
        </div>
        <div id="searchResult"></div>
    </div>
</div>

<script src="/js/search.js"></script>
<script type="text/javascript">
$(function () {
    searchFunc("/" + "search.xml", 'searchInput', 'searchResult');
});
</script>
    <!-- 回到顶部按钮 -->
<div id="backTop" class="top-scroll">
    <a class="btn-floating btn-large waves-effect waves-light" href="#!">
        <i class="fas fa-arrow-up"></i>
    </a>
</div>


    <script src="/libs/materialize/materialize.min.js"></script>
    <script src="/libs/masonry/masonry.pkgd.min.js"></script>
    <script src="/libs/aos/aos.js"></script>
    <script src="/libs/scrollprogress/scrollProgress.min.js"></script>
    <script src="/libs/lightGallery/js/lightgallery-all.min.js"></script>
    <script src="/js/matery.js"></script>

    <!-- Global site tag (gtag.js) - Google Analytics -->


    <!-- Baidu Analytics -->


    <!-- Baidu Push -->

<script>
    (function () {
        var bp = document.createElement('script');
        var curProtocol = window.location.protocol.split(':')[0];
        if (curProtocol === 'https') {
            bp.src = 'https://zz.bdstatic.com/linksubmit/push.js';
        } else {
            bp.src = 'http://push.zhanzhang.baidu.com/push.js';
        }
        var s = document.getElementsByTagName("script")[0];
        s.parentNode.insertBefore(bp, s);
    })();
</script>

    
    <script src="/libs/others/clicklove.js" async="async"></script>
    
    

    

    
    <script>
        (function (i, s, o, g, r, a, m) {
            i["DaoVoiceObject"] = r;
            i[r] = i[r] || function () {
                (i[r].q = i[r].q || []).push(arguments)
            }, i[r].l = 1 * new Date();
            a = s.createElement(o), m = s.getElementsByTagName(o)[0];
            a.async = 1;
            a.src = g;
            a.charset = "utf-8";
            m.parentNode.insertBefore(a, m)
        })(window, document, "script", ('https:' == document.location.protocol ? 'https:' : 'http:') +
            "//widget.daovoice.io/widget/6984b559.js", "daovoice")
        daovoice('init', {
            app_id: ""
        });
        daovoice('update');
    </script>
    

    

    

    
    <script type="text/javascript" src="/libs/background/ribbon-dynamic.js" async="async"></script>
    
    
    
    <script src="/libs/instantpage/instantpage.js" type="module"></script>
    

</body>

</html>
